src/lib/modules/Items/backoffice/ItemListEntry.js

```diff
@@ -78,22 +78,33 @@ export class ItemListEntry extends Component {
             <Grid.Column computer={6} largeScreen={6}>
               <Item.Meta className="metadata-fields">
                 <Header as="h5">
+                <Link
```

Comment:
how this influence the Items search view ? did you have any notification of changed snaphots in ItemSearch (in tests)?

Reply:
I didn't, it seems to work fine, but I will double-check.